src/checker.ml

```diff
@@ -615,12 +616,15 @@ let entrypoint_remove_liquidity (state, p: checker * (lqt * ctez * kit * Ligo.ti
 (**                          LIQUIDATION AUCTIONS                            *)
 (* ************************************************************************* *)
 
-let entrypoint_liquidation_auction_place_bid (state, kit: checker * kit) : LigoOp.operation list * checker =
+let entrypoint_liquidation_auction_place_bid (state, (auction_id, kit): checker * (Ligo.nat * kit)) : LigoOp.operation list * checker =
   assert_checker_invariants state;
   let _ = ensure_no_tez_given () in
 
   let bid = { address=(!Ligo.Tezos.sender); kit=kit; } in
   let current_auction = liquidation_auction_get_current_auction state.liquidation_auctions in
+  let () = if nat_of_ptr (ptr_of_avl_ptr current_auction.contents) = auction_id
```

Comment:
I initially did that, but then I thought that I'd prefer to have a `nat` on the endpoint and making the conversion explicit (shouldn't introduce a cost since it's all single constructor wrappers). But I do not feel strongly here, I'll do as you suggested.